### PR TITLE
Release 0.20.0-alpha5

### LINKED ISSUE #1313: Release alpha5 with parking_lot bump?
I'm hoping to get the ecosystem moved to the newer parking_lot and smallvec. Could a new alpha be released for winit (and hopefully glutin too?)

### PR DESCRIPTION
**Pull request by Manishearth**
Fixes #1313

r? @goddessfreya

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Unreleased
+# Unrealeased
```
Comment (simlay) [inline CHANGELOG.md]: Typo?

Comment (Manishearth) [reply CHANGELOG.md]: Fixed

**Comment by goddessfreya** [comment]
Ty!

**Comment by Manishearth** [comment]
Is there a rough timeline on when glutin and winit will have a non-alpha release?

**Comment by goddessfreya** [comment]
We are waiting on the last couple of platforms for these issues: https://github.com/rust-windowing/winit/labels/Blocking%20a%20release

**Comment by Manishearth** [comment]
Thank you!

**Pull request merged**